DuckDuckGoTests/UserAgentTests.swift

```diff
@@ -28,6 +28,8 @@ final class UserAgentTests: XCTestCase {
     private struct DefaultAgent {
         static let mobile = "Mozilla/5.0 (iPhone; CPU iPhone OS 12_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148"
         static let tablet = "Mozilla/5.0 (iPad; CPU OS 12_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148"
+        static let oldWebkitVersionMobile = "Mozilla/5.0 (iPhone; CPU iPhone OS 12_4 like Mac OS X) AppleWebKit/605.1.14 (KHTML, like Gecko) Mobile/15E148"
+        static let newWebkitVersionMobile = "Mozilla/5.0 (iPhone; CPU iPhone OS 12_4 like Mac OS X) AppleWebKit/605.1.16 (KHTML, like Gecko) Mobile/15E148"
```

Comment:
Can you also add a test for `AppleWebKit/605.1.15` too? I'm a little nervous based on that comparison

Reply:
of course, I didn't add it initially, because pretty much everything else testes this use case